Gems/LyShine/Code/Editor/SpriteBorderEditorCommon.h

```diff
@@ -10,7 +10,6 @@
 #include <vector> // required to be included before platform.h
 #include <platform.h>
 #include <ISystem.h>
```

Comment:
Do you have plan to remove the dependency of CrySystem? 
I think AZCore should provide everything you may need to replace them.

Reply:
Yes, CrySystem dependency will be addressed separately.